README.md

```diff
@@ -1,264 +1,505 @@
+# NAMETBD: A design system integrating to Pattern Lab and a Drupal 8 theme
+
 [![Build Status](https://travis-ci.org/phase2/pattern-lab-starter.svg?branch=master)](https://travis-ci.org/phase2/pattern-lab-starter)
 
-# Prerequisites
+NAMETBD is an opinionated set of tools and examples to:
+
+1. Build an application-agnostic **design system**
+1. Apply that design system to a locally-served **Pattern Lab** for rapid prototyping
+1. Apply that design system to a **Drupal theme**
```

Comment:
For pure markdown reading, I find it nicer to use 1/2/3 for bullets, all markdown cares about is `\d+\.`

Does bullet 1 sufficiently capture "establish a framework to support more application frameworks?"

Reply:
I'm running against a "markdown linter" and the use-case is for easier re-arranging at the text level, given that the rendered-level is always ordered.